### DISCUSSION TITLE
Allow custom subject lines for new user invites

### DISCUSSION THREAD

**Discussion by JasonAGross** [directus/directus#17727] [Feature Requests]
### Summary

The `inviteUser` service already accepts `subject` as an optional field, however the controller schema will error out if you try to pass a subject into the `/users/invite` endpoint. The change to allow for custom subject lines is straightforward and I see no reason why it shouldn't be supported. 

### Basic Example

A call to the invite API could simply be modified from this: 

```
await api.post('/users/invite', {
	email: emailsParsed,
	role: selectedRole,
});
```
to: 
```
await api.post('/users/invite', {
	email: emailsParsed,
	role: selectedRole,
        subject: 'My custom subject line',
});
```

### Motivation

We have a custom implementation of the user invite feature to automatically associate new users with a specific set of content within our extensions. We would like to include additional information in the email subject.

### Detailed Design

This is very straightforward, the support for a custom subject is already there, we just need to allow for it to be passed through the controller. 

### Must Have:
- Custom email subjects for new user invites

### Drawbacks

Subject can be treated as an optional field, allowing for backwards compatibility. The existing implementation of the user invite feature can remain unchanged. 

### Alternatives

N/A

### Adoption Strategy

If other users want to utilize the invite endpoint in their extensions they can now include a subject. This will be inline with how something like the notifications service is already used. Adoption should be straightforward.

**Comment by JasonAGross** [comment]
PR for the change: https://github.com/directus/directus/pull/17728

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
